src/component/user/user.scss

```diff
@@ -2,4 +2,14 @@
     border-radius: 25px;
     height: 32px;
     border: 2px solid #ffffff;
-}
\ No newline at end of file
+}
+
+.showLocale img {
+    border-radius: 2px;
+    height: 20px;
+    border: 2px solid #ffffff;
+}
```

Comment:
How about dropping border, increasing height to 30px and adding some margin? 

```css
    border-radius: 2px;
    height: 30px;
    margin: 0 10px;
```

![screenshot from 2018-02-08 20-37-43](https://user-images.githubusercontent.com/158948/35994239-194cc546-0d10-11e8-9434-709535e718d1.png)


(PS! I am no UX expert)